crates/bindings/tests/snapshots/deps__spacetimedb_bindings_dependencies.snap

```diff
@@ -4,12 +4,20 @@ expression: "cargo tree -p spacetimedb -f {lib} -e no-dev"
 ---
 total crates: 61
 spacetimedb
+├── bytemuck
+│   └── bytemuck_derive
```

Comment:
Maybe disable default-features of bytemuck, so that it doesn't pull in the derive?